giraffe/src/utils/fromFlux.ts

```diff
@@ -189,6 +189,151 @@ export const fromFlux = (fluxCSV: string): FromFluxResult => {
   }
 }
 
+export const fastFromFlux = (fluxCSV: string): FromFluxResult => {
+  const columns: Columns = {}
+  const fluxGroupKeyUnion = new Set<string>()
+  const resultColumnNames = new Set<string>()
+  let tableLength = 0
+
+  try {
+    const chunks = splitChunksFast(fluxCSV)
```

Comment:
pull this out of the split function and back into the main function so you dont have the function overhead of passing these chunks around

Reply:
Gotcha